fix_test.go

```diff
@@ -23,11 +24,14 @@ func TestFix(t *testing.T) {
 		{"all-resources_v1beta2.yml", "all-resources-fixed_v1beta2.yml"},
 	}
 
+	allAuditors, err := all.Auditors(config.KubeauditConfig{})
+	assert.NoError(t, err)
```

Comment:
If you're depending on this further down in the tests, it might make more sense to use `require.NoError` instead of `assert.NoError`